pos_partner_firstname/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Roberto Fichera
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.model
+    def create_from_ui(self, partner):
+        if 'is_company' in partner:
+            partner['is_company'] = partner['is_company'] == 'true'
+        if 'electronic_invoice_subjected' in partner:
```

Comment:
This part should stay in the Italian module

Reply:
@eLBati Right! But only the part related to ```electronic_invoice_subjected``` need to be removed, the rest is necessary.